### DISCUSSION TITLE
Tag of a new version

### DISCUSSION THREAD

**Discussion by sebastienraillet** [COVESA/dlt-daemon#450] [Q&A] [answered]
Hello all 👋 
The question is mainly addressed to dlt-daemon maintainers (@michael-methner and @lvklevankhanh).
It is planned to freeze / tag a new version in a near future ? The last tag [v2.18.8](https://github.com/COVESA/dlt-daemon/releases/tag/v2.18.8) has been done in October 2021.
Lot of amazing work has been performed since 😉 I can, for sure, use the master branch to have a fresh version but having a reference point frozen in the time is (of my point of view) something useful.
Thanks,

**Comment by michael-methner** [comment, marked as answer]
Hello @sebastienraillet ,
actually we already planned for quite a while for a new release but we have a set of patches internally which we would like to upstream before. 

So I can't promise any date yet for a v2.18.9 release but I expect it to happen in April to May 2023.

**Reply by michael-methner** [reply]
Hello @sebastienraillet ,
we have decided to make a prerelease first due to the amount of changes happened since the last release. We will use the next weeks to stabilize this release and make an official release afterwards.

We plan to make release in the future more regular.

**Reply by sebastienraillet** [reply]
Hello @michael-methner,
Thanks for the news 🙂
